src/dataflow/src/sink/kafka.rs

```diff
@@ -62,7 +62,7 @@ use mz_timely_util::operators_async_ext::OperatorBuilderExt;
 
 use super::KafkaBaseMetrics;
 use crate::render::sinks::SinkRender;
-use prometheus::core::{AtomicI64, AtomicU64};
+use prometheus::core::{AtomicI64 as PrometheusAtomicI64, AtomicU64 as PrometheusAtomicU64};
```

Comment:
These atomic integers default to using `Relaxed` ordering so I've made them the non-standard imports